cartographer_ros_msgs/msg/metrics/HistogramBucket.msg

```diff
@@ -0,0 +1,16 @@
+# 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+float64 bucket_boundary
```

Comment:
Can boundaries be different within a family?

Reply:
Not by definition, but the new generic Metric msg format that Christoph proposed also uses `cartographer_ros_msgs/HistogramBucket[] counts_by_bucket`. Moving the buckets into the MetricFamily.msg would require another type differentiation there without much benefit in my opinion.